### PR TITLE
Add Eastern Bank

### PR DESCRIPTION
**Pull request by lazytownfan**
* This pull request adds Eastern Bank
* I was only able to find a PNG of the logo
* There is currently no 2FA methods for Eastern Bank

### CODE REVIEW

File: entries/e/easternbank.com.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "Eastern Bank": {
+    "domain": "easternbank.com",
+    "url": "https://www.easternbank.com/",
```
Comment (hkamran80) [inline entries/e/easternbank.com.json]:
```suggestion
```

The `url` key is only needed if going to the root domain (e.g. `https://easternbank.com`) fails

**Review by hkamran80 — CHANGES_REQUESTED** [review]
Please rename the PNG to `easternbank.com.png` to match the other site logos, and also run the PNG through [TinyPNG](https://tinypng.com/).

**Comment by phallobst** [comment]
Here's an SVG version of the logo:
easternbank com

**Comment by lazytownfan** [comment]
Thank you @phallobst for the SVG!

File: entries/e/easternbank.com.json (hunk 1 above)
Comment (lazytownfan) [reply entries/e/easternbank.com.json]: * I removed the `url` key.
* I had already ran the PNG through TinyPNG, but @phallobst already (and graciously) gave me the SVG version of the logo.  (Thank you again!)
    * I am now using the SVG image.

File: entries/e/easternbank.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "Eastern Bank": {
+    "domain": "easternbank.com",
+    "contact": {
+      "facebook": "easternbank",
+      "twitter": "easternbank",
+      "form": "https://www.easternbank.com/form/customer-feedback"
+    },
```
Comment (phallobst) [inline entries/e/easternbank.com.json:8]:
```suggestion
    },
    "regions": [
      "us"
    ],
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by hkamran80 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**